gb-ppu/src/memory/vram.rs

```diff
@@ -28,6 +28,21 @@ impl From<BankSelector> for usize {
     }
 }
 
+impl TryFrom<u8> for BankSelector {
```

Comment:
Isn't this implementation conflicting with `Vbk => self.vbk.set(v | VBK_UNUSED_BITS),` (the line where you set the `VBK` reg) ?

Reply:
You are right, this will cause a panic. On this part I forgot that all unused bits are 1.